blockly-core/core/ui/css.js

```diff
@@ -80,50 +79,14 @@ Blockly.Css.setCursor = function(cursor, opt_svg) {
     return;
   }
 
-  if (Blockly.Css.currentCursor_ != cursor) {
-    Blockly.Css.currentCursor_ = cursor;
-
-    /*
-    Hotspot coordinates are baked into the CUR file, but they are still
-    required in the CSS due to a Chrome bug.
-    https://code.google.com/p/chromium/issues/detail?id=1446
-    */
-    if (cursor == Blockly.Css.Cursor.OPEN) {
-      var xy = '8 5';
-    } else {
-      var xy = '7 3';
-    }
-    var cursorRuleRHS = 'url(' +
-    Blockly.assetUrl('media/' + cursor + '.cur') +
-    ') ' + xy + ', auto';
-    var rule = '.blocklyDraggable {\ncursor: ' + cursorRuleRHS + ';\n}\n';
-    var ruleIndex = 0;
-    // Guard against empty stylesheet for tests.
-    if (Blockly.Css.styleSheet_ && Blockly.Css.styleSheet_.cssRules.length > ruleIndex) {
-      // There are potentially hundreds of draggable objects.  Changing their style
-      // properties individually is too slow, so change the CSS rule instead.
-      goog.cssom.replaceCssRule('', rule, Blockly.Css.styleSheet_, ruleIndex);
-    }
-  }
-
-  var setCursorOnBackgroundElement = function(element) {
-    if (cursor == Blockly.Css.Cursor.OPEN) {
-      element.style.cursor = '';
-    } else {
-      element.style.cursor = cursorRuleRHS;
-    }
-  };
-
-  // There is probably only one toolbox, so just change its style property.
-  var toolboxen = document.getElementsByClassName('blocklyToolboxDiv');
-  for (var i = 0, toolbox; toolbox = toolboxen[i]; i++) {
-    setCursorOnBackgroundElement(toolbox);
```

Comment:
It looks like this code to set the cursor style on the toolboxe(s|n) is not taken into account in your refactor.  I think this is in place to minimize cursor flicker (like why we change the cursor rule for the workspace background).  This may be as simple as adding/removing the `.dragging` class for all `.blocklyToolboxDiv` elements too.